src/services/supabase.ts

```diff
@@ -370,7 +323,7 @@ export const updateSupabase = (
     matchData: any
 ): PromiseLike<CallSupabaseResponse<any>> => {
     return supabaseRetry(
-        () => supabaseClient.from(table).update(data).match(matchData),
+        () => supabaseClient.from(table).update(data).match(matchData).select(),
```

Comment:
we have a lot of code that expect updates to the db to return what they changed.